README.md

```diff
@@ -142,7 +169,10 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+Schema defines what kinds of data database can accept, integer, string, etc
+Bipin has many wishes which are stored on his wishes table. wishes table also has ID associated with each wish rows. Wishes table might have wishes that belong to Alladin.
+
+
 ```
```

Comment:
A schema file not only defines the data types that we'll be storing in a database, but it also defines the column names as well.